config/jazzy/requirements/features-executable.yaml

```diff
@@ -362,11 +362,11 @@ requirements:
     checks:
       - name: >-
           Check `gazebo --verbose
-          /opt/ros/rolling/share/gazebo_plugins/worlds/gazebo_ros_camera_distortion_barrel_demo.world`
+          /opt/ros/jazzy/share/gazebo_plugins/worlds/gazebo_ros_camera_distortion_barrel_demo.world`
```

Comment:
You might also want to go through all the requirements and update the gazebo classic references to gazebo (gz).

Reply:
Yeah, that's an excellent point.  We'll definitely need to do that.  Maybe @azeey can help here?

Reply:
Great point, thanks! will merge this first and we can do that on a separate PR.

Reply:
I started looking at this but I was confused about the command `ros2 run gazebo_ros_pkgs gazebo`. Did that ever work? I don't see any executables in the [package](https://github.com/ros-simulation/gazebo_ros_pkgs/tree/ros2/gazebo_ros_pkgs). 

Also, is this meant to just check that Gazebo is working? i.e, can I just use a simple world that gets shipped with Gazebo?